### DISCUSSION TITLE
vercel部署ui出错

### DISCUSSION THREAD

**Discussion by zy111x** [excalidraw/excalidraw#8734] [General]
![Image](https://github.com/user-attachments/assets/20c313fe-fb3a-4e62-9f8e-7cdfe70f9f4c)
为什么我在vercel上部署的ui界面是这样？

**Comment by Mrazator** [comment]
> Why is the UI interface I deployed on Vercel like this?

Looks like what you deployed is our nextjs example: https://github.com/excalidraw/excalidraw/tree/master/examples/excalidraw/with-nextjs

**Reply by wibur0620** [reply]
> > Why is the UI interface I deployed on Vercel like this?
> 
> Looks like what you deployed is our nextjs example: https://github.com/excalidraw/excalidraw/tree/master/examples/excalidraw/with-nextjs

I have the same problem as him, I read the link you replied to and still don't understand how to solve this problem, is this ui interface supposed to be correct? Or is there any way to make the ui interface normal?

**Reply by narallee** [reply]
When you deploy in Vercel, you need change the **Root Directory** to empty , default is under the examples folder.

**Comment by acshameless** [comment]
using vite instead of next.js

**Comment by aric6494** [comment]
For me, I followed these steps:

1. Fork Excalidraw to your GitHub account.
2. Delete the entire **examples directory**.
![image](https://github.com/user-attachments/assets/62a5ad74-68f1-4119-b8ba-afdbb6578ef2)

3. Finally, import the forked repository into Vercel and deploy.
